chatcommands.py

```diff
@@ -1,6 +1,7 @@
 # coding=utf-8
 # noinspection PyUnresolvedReferences
-from chatcommunicate import add_room, block_room, CmdException, command, get_report_data, is_privileged, message, \
+from chatcommunicate import add_room, block_room, CmdException, command, get_report_data, \
+    is_privileged, message, \
```

Comment:
Based on this change and another change below, I assume your IDE doesn't like longer lines. I mention this merely because this being changed resulted in me spending time trying to determine if there was a functional change in the lines (i.e. visually comparing each portion of the line, not finding one and then going back and checking it again). That's not a problem, I'm just wanting to be sure I'm not missing a functional change.

Reply:
Correct, IDE enforces PEP8 + 120char maxlimit per line